minsik/db/migrations/20221123070801_users.sql

```diff
@@ -0,0 +1,15 @@
+-- migrate:up
+CREATE TABLE users (
+    id INT NOT NULL AUTO_INCREMENT,
+    name VARCHAR(50) NOT NULL,
+    email VARCHAR(200) NOT NULL,
+    profile_image VARCHAR(1000) NULL,
+    password VARCHAR(200) NOT NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
+    PRIMARY KEY(id),
+    UNIQUE (email)
```

Comment:
```suggestion
    ATLER TABLE users ADD CONSTRAINT users_email_unique UNIQUE (email);
```

위의 코드와 같이 제약 조건을 지정해 줄 수도 있을 것 같습니다!